pack/tdc/tdcp.json

```diff
@@ -281,6 +286,7 @@
         "position": 9,
         "quantity": 1,
         "restrictions": "investigator:11007, investigator:11008",
+        "signature_for": "11007",
```

Comment:
If the intent is to separate the two Agathas - which I agree with - then this should probably be `"signature_for": ["11007", "11008"]` or `"signature_for": "11007, 11008"`.

This _could_ be relevant for Chapter 2 if Replacements are ruled to affect all versions of an investigator.